static/js/click_handlers.js

```diff
@@ -242,11 +242,6 @@ export function initialize() {
     });
 
     $("body").on("click", ".reveal_hidden_message", (e) => {
-        // Hide actions popover to keep its options
-        // in sync with revealed/hidden state of
-        // muted user's message.
-        popovers.hide_actions_popover();
-
```

Comment:
What's the story with this removal? Could imagine this being its own commit with its own commit message if appropriate.

Reply:
We still need to hide the actions popover after porting it to tippy, but clicking on the message hides the action popover 
automatically, so this just needs to be removed in the commit we port actions popover to tippy. So, we don't need a different commit for this.

Reply:
OK. As a note on good strategies for asking for review, posting PR comments on details like this where it's not a direct translation/move and thus some explanation is required to verify it's correct can save review round-trips. (This sort of thing isn't a good case for a code comment, because no special note should be required to read the post-change code).

Reply:
Okay! Thanks!